meadow/src/Git.hs

```diff
@@ -1,4 +1,28 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE TemplateHaskell #-}
 module Git where
 
-gitHead :: String
-gitHead = "master"
+import qualified Data.Text as T
+import Data.Text (Text)
+import Data.Text.Encoding (decodeUtf8)
+import Data.String (fromString)
+import           Data.FileEmbed (dummySpaceWith)
+import Git.TH (gitRevFromGit)
+
+gitRev :: Text
+gitRev | gitRevEmbed /= zeroRev = gitRevEmbed
+       | T.null fromGit         = zeroRev
+       | otherwise              = fromGit
```

Comment:
Why do we need this?

Reply:
it just does best efforts getting the revision, this means it works with `stack build` which is nice.